src/vellum/workflows/__init__.py

```diff
@@ -1,5 +1,67 @@
-from .workflows import BaseWorkflow
+import os
+from uuid import uuid4
+from typing import Optional
+
+from .context import ExecutionContext
+from .state.context import WorkflowContext
+from .workflows.base import BaseWorkflow
+
+
+def enable_workflow_monitoring(api_key: Optional[str] = None, api_url: Optional[str] = None) -> WorkflowContext:
```

Comment:
[blocking] I think we could remove this for now. We imagine a similar feature being built through @marlenc3's otel work. for vembda service, we are already constructing a `WorkflowContext` that we'll simply add a `VellumEmitter` to